cmd/helm/serve.go

```diff
@@ -55,19 +55,29 @@ func newServeCmd(out io.Writer) *cobra.Command {
 		Use:   "serve",
 		Short: "start a local http web server",
 		Long:  serveDesc,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return srv.complete(args)
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return srv.run()
 		},
 	}
 
 	f := cmd.Flags()
-	f.StringVar(&srv.repoPath, "repo-path", settings.Home.LocalRepository(), "local directory path from which to serve charts")
+	f.StringVar(&srv.repoPath, "repo-path", "", "local directory path from which to serve charts")
```

Comment:
Working on a better fix for setting the default path here, but want to fix the bigger bug first.